INN/umbrella-inndev
# Increase startup guide image field size

## Changes

This pull request makes the following changes:

- Adjusted `min-width` attribute of `#biggo` homepage section from 200px to 240px so the image size matches the size of the images below it.

## Why



For #112 

## Testing/Questions

Features that this PR affects:

- Homepage icons

Steps to test this PR:

1. View the homepage and verify the top icon is the same size as the other icons
